calcom/cal.com
# fix: updated the bg-emphasis style in dark mode

## What does this PR do?



I have changed the color of --cal-bg-emphasis of dark mode in global.css from #525252 to #404040.

Fixes #12888



## Type of change




## How should this be tested?



You've to turn on the dark mode of your system or you can turn on the dark mode in cal also by navigating to the setting > appearance. Then you have to check the color of the active item of sidebar menu.
